Spanner/src/Connection/Grpc.php

```diff
@@ -69,6 +69,16 @@
 use Google\Protobuf\Timestamp;
 use GuzzleHttp\Promise\PromiseInterface;
 
+class InsecureCredentialsWrapper extends CredentialsWrapper
+{
+    function __construct() {}
+
+    public function getAuthorizationHeaderCallback($audience = null)
+    {
+        return null;
+    }
+}
```

Comment:
This will need to be moved into its own class file. I recommend putting it in the [Core](https://github.com/googleapis/google-cloud-php/tree/main/Core) directory (e.g. `Core/InsecureCredentialsWrapper.php`), as this can be used by all the various packages which use the emulator (Bigtable, Datastore, Firestore, Pubsub, and Spanner)

Reply:
I did have the very same issue with unauthenticated errors with Firestore as well. Would you mind fixing it in `Google\Cloud\Firestore\Connection\Grpc` class as well? Seems similar enough to Spanner's case.

Reply:
yes, I will

Reply:
review comment addressed